src/libraries/Directory.Build.targets

```diff
@@ -118,8 +118,7 @@
   <Import Project="$(RepositoryEngineeringDir)testing\coverage.targets" Condition="'$(EnableCoverageSupport)' == 'true'" />
 
   <Import Sdk="Microsoft.DotNet.Build.Tasks.TargetFramework.Sdk" Project="Sdk.targets" />
-  <Import Condition="'$(EnableProjectRestore)' != 'true'" Project="$(RepositoryEngineeringDir)restore\repoRestore.targets" />
-
+  <Import Project="$(RepositoryEngineeringDir)restore\repoRestore.targets" Condition="'$(DisableProjectRestore)' == 'true'" />
```

Comment:
Why are we importing the `repoRestore.targets` when `DisableProjectRestore` is set to true? That is confusing, is it intentional?

Reply:
DisableProjectRestore means that the project's restore target is a no-op. `repoRestore` means that everything comes via the Tools.props Arcade restore import. I don't think we need to change that given that we will remove `repoRestore.props` and `repoRestore.targets` and remove that property when depprojs are removed, anyway. That should happen soon-ish.

Reply:
Ok, it is just confusing to have a Condition to disable restore and import targets that are named repoRestore.